erpnext/accounts/report/accounts_receivable/accounts_receivable.py

```diff
@@ -802,8 +787,7 @@ def setup_ageing_columns(self):
 		for i, label in enumerate(["0-{range1}".format(range1=self.filters["range1"]),
 			"{range1}-{range2}".format(range1=cint(self.filters["range1"])+ 1, range2=self.filters["range2"]),
 			"{range2}-{range3}".format(range2=cint(self.filters["range2"])+ 1, range3=self.filters["range3"]),
-			"{range3}-{range4}".format(range3=cint(self.filters["range3"])+ 1, range4=self.filters["range4"]),
-			"{range4}-{above}".format(range4=cint(self.filters["range4"])+ 1, above=_("Above"))]):
```

Comment:
@rvpasari  Since you are removing the range4 from the server script. let's remove it from the client script too. otherwise its unused code in the client script.

Reply:
We can discuss this issue in morning standup